src/transformers/models/oneformer/image_processing_oneformer.py

```diff
@@ -347,6 +347,22 @@ def prepare_metadata(repo_path, class_info_file):
     return metadata
 
 
+def load_metadata(repo_id, class_info_file):
+    fname = os.path.join("" if repo_id is None else repo_id, class_info_file)
+
+    if not os.path.exists(fname) or not os.path.isfile(fname):
+        # We try downloading from a dataset by default for backward compatibility
+        try:
+            fname = hf_hub_download(repo_id, class_info_file, repo_type="dataset")
+        except RepositoryNotFoundError:
+            fname = hf_hub_download(repo_id, class_info_file)
```

Comment:
any reason to try without `repo_type=dataset`?

Reply:
Just that someone might store it under the model repo rather than the dataset. As the model is trained on a dataset - there's already a coupling and we already store data specific mappings e.g. id2label on the model side. 

I can remove if you think it's better?

Reply:
It's ok :-)